src/pages/workspaces/workspace/notebooks/AnalysisLauncher.js

```diff
@@ -520,13 +550,12 @@ const WelderDisabledNotebookEditorFrame = ({
       notify('error', 'Cannot Edit Notebook', {
         message: h(Fragment, [
           p(['Recent updates to Terra are not compatible with the older cloud environment in this workspace. Please recreate your cloud environment in order to access Edit Mode for this notebook.']),
-          h(Link, { href: dataSyncingDocUrl, ...Utils.newTabLinkProps }, ['Read here for more details.'])
+          h(Link, { 'aria-label': 'Data syncing doc', href: dataSyncingDocUrl, ...Utils.newTabLinkProps }, ['Read here for more details.'])
```

Comment:
I think, in context, it makes sense to say 'read more here' rather than the name of a doc, since that's the action we want people to do.
```suggestion
          h(Link, { href: dataSyncingDocUrl, ...Utils.newTabLinkProps }, ['Read here for more details.'])
```